src/components/about/Founders.tsx

```diff
@@ -31,10 +35,10 @@ const Founder = ({founder}: Founder) => {
   )
 }
 
-const SocialIcon = ({social}) => {
+const SocialIcon = ({social}: SocialIconProps) => {
   const [isHovered, setIsHovered] = useState(false)
 
-  const getIcon = (name, isHovered) => {
+  const getIcon = (name: string, isHovered: boolean) => {
     switch (name) {
       case "linkedin":
         return (
```

Comment:
should this be read from files ?

Reply:
Do you mean why we are rendering this here instead of adding the icon in the icons folder? @amitksingh1490